libraries/botframework-streaming/src/utilities/createNodeServer.ts

```diff
@@ -0,0 +1,30 @@
+import { INodeSocket2 } from "../interfaces/INodeSocket2";
+
+/**
+ * @module botframework-streaming
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+ // What type hint should I put, if not importing 'net'?
+export const createNodeServer = function(callback?: (socket: INodeSocket2) => void) {
+    if (callback && typeof callback !== 'function') {
+        throw new TypeError(`Invalid callback; callback parameter must be a function to create Node 'net' Server.`);
+    }
+
+    return getNetServerConstructor()(callback);
+}
+
+export const getNetServerConstructor = function() {
```

Comment:
naming nit: a constructor is usually  `new Object` pattern. This is a factory method.

Reply:
You also need to assert the type signature here, I think. Else, you might run into issues with the types being output here. Check the .d.ts here to make sure it isn't outputting the `require('net').Server` signature.